paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/networking/Models.kt

```diff
@@ -141,7 +142,7 @@ data class ExampleCreateAndConfirmErrorResponse(
                     serializer(),
                     body
                 )
-            } catch (ex: Exception) {
+            } catch (@Suppress("SwallowedException") ex: SerializationException) {
```

Comment:
I think you want to catch more. What if the response body is null, or not of type json?

Reply:
Ok, I will just catch all.